OpenCL_LangExt.txt

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 The Khronos Group. This work is licensed under a
+// Creative Commons Attribution 4.0 International License; see
+// http://creativecommons.org/licenses/by/4.0/
+
+= The OpenCL^(TM)^ Language Extensions Specification
+:R: pass:q,r[^(R)^]
+Khronos{R} OpenCL Working Group
+:data-uri:
+:icons: font
+:toc2:
+:toclevels: 2
+:max-width: 100
+:numbered:
+:imagewidth: 800
+:fullimagewidth: width="800"
+:source-highlighter: coderay
+:title-logo-image: image:images/OpenCL.png[Logo,200,200]
+
+:blank: pass:[ +]
+:cpp: pass:[C++]
+:cpp14: pass:[C++14]
+
+// Various special / math symbols. This is easier to edit with than Unicode.
+include::config/attribs.txt[]
+
+// type of the source code in the document
+:language: {basebackend@docbook:c++:cpp}
+
+include::copyrights.txt[]
```

Comment:
Confirming: This document shouldn't have the Khronos spec copyright since it is not a ratified Khronos specification?